### PR TITLE
fixes for smi

### PR DESCRIPTION
**Pull request by kumarabd**
Signed-off-by: kumarabd <abishekkumar92@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,17 +2,21 @@ module github.com/layer5io/meshery-linkerd
...
 	google.golang.org/grpc v1.32.0
-	k8s.io/api v0.17.4
-	k8s.io/apiextensions-apiserver v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/client-go v0.17.4
+	k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery v0.18.8
+	k8s.io/client-go v0.18.8
 	k8s.io/kube-aggregator v0.17.4
```
Comment (Aisuko) [inline go.mod:21]: If we already to upgrade the `k8s.io/xxx` version, we should upgrade them all at once time. If do not, may bring other issues

File: linkerd/smi_conformance.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/layer5io/gokit/smi"
+	"github.com/layer5io/meshery-linkerd/meshes"
+)
+
+func (iClient *Client) runConformanceTest(id string, name string, version string) error {
+
```
Comment (Aisuko) [inline linkerd/smi_conformance.go:13]: I have already add `golangci-lint` to check these issues, but we need to merge the PR first

**Review by Aisuko — CHANGES_REQUESTED** [review]
A little advices, please check below

File: go.mod (hunk 1 above)
Comment (kumarabd) [reply go.mod:21]: @Aisuko yes you are right. I have made the changes for all of those functions. Can you review them once?

File: linkerd/smi_conformance.go (hunk 2 above)
Comment (kumarabd) [reply linkerd/smi_conformance.go:13]: @Aisuko  I have merged that. Lets check this now?

File: go.mod (hunk 1 above)
Comment (Aisuko) [reply go.mod:21]: We need fix some conflict

**Comment by leecalcote** [comment]
One failing check - https://github.com/layer5io/meshery-linkerd/pull/138/checks?check_run_id=1349099257#step:4:26

**Comment by kumarabd** [comment]
> One failing check - https://github.com/layer5io/meshery-linkerd/pull/138/checks?check_run_id=1349099257#step:4:26
> 
> @kumarabd maybe we can use `make golangci-lint` execution at local environment, may be it can help you out.
> 
> https://github.com/layer5io/meshery-linkerd/blob/f2aff92347096940739df3a546b684883e14fe1e/Makefile#L32-L38

Yes @leecalcote I'm currently using that.

**Pull request merged**